R/zchunk_LB125.LC_tot.R

```diff
@@ -65,38 +64,132 @@ module_aglu_LB125.LC_tot_DISABLED <- function(command, ...) {
     #
     # ===================================================
 
+    # -----------------------------------------------------------------------------
+    # 2. Perform computations
+    #Create a table with all land types
+    #NOTE: If protected lands are included, uncomment the text below
```

Comment:
No--just make changes to your branch, and when you commit and push them, the PR here will be automatically updated.